test/e2e/far_e2e_test.go

```diff
@@ -405,7 +405,7 @@ func wasNodeRebooted(nodeName string, nodeBootTimeBefore time.Time) {
 
 // checkVaDeleted verifies if the va has already been deleted due to resource deletion
 func checkVaDeleted(va *storagev1.VolumeAttachment) {
-	EventuallyWithOffset(1, func() bool {
+	ConsistentlyWithOffset(1, func() bool {
```

Comment:
Why this change? For waiting until something happens, Eventually is the right tool 🤔